### PR TITLE
Adding Hostname:IP address qualifier into bigdime_alerts column family

### PR DESCRIPTION
**Pull request by sandeepreddymurthy1**
Integration test for Put

### CODE REVIEW

File: bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java [hunk 1]
```diff
@@ -65,12 +71,39 @@
...
+			Enumeration e = NetworkInterface.getNetworkInterfaces();
+			while(e.hasMoreElements())
+			{
+			    NetworkInterface n = (NetworkInterface) e.nextElement();
+			    Enumeration ee = n.getInetAddresses();
+			    while (ee.hasMoreElements())
+			    {
+			        InetAddress i = (InetAddress) ee.nextElement();
+			        if(i.getHostAddress().startsWith(IP_INIT_VAL)){
+			        	 System.out.println(i.getHostAddress());
```
Comment (naithania) [inline bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java]: remove system.

File: bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java [hunk 2]
```diff
@@ -252,6 +285,7 @@ private Put buildPut(String source, final String shortMessage, final String mess
 		Put put = new Put(createRowKey(source));
 		addToPut(put, ALERT_ADAPTOR_NAME_COLUMN, source);
 		addToPut(put, ALERT_MESSAGE_CONTEXT_COLUMN, shortMessage);
+		addToPut(put, ALERT_HOST_DETAILS, hostName+":"+hostIp);
```
Comment (naithania) [inline bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java]: declare variable

File: bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java (hunk 1 above)
Comment (sandeepreddymurthy1) [inline bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java]: System.out.println removed

File: bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java (hunk 2 above)
Comment (sandeepreddymurthy1) [inline bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java]: host ip has been added to new a new qualifier.No need of the varable now

File: bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java [hunk 3]
```diff
@@ -65,12 +72,39 @@
...
+			    {
+			        InetAddress i = (InetAddress) ee.nextElement();
+			        if(i.getHostAddress().startsWith(IP_INIT_VAL)){
+			        	 hostIp=i.getHostAddress();
+			        	 break;
+			        }
+			    }
+			}
+		} catch (UnknownHostException e) {
+			e.printStackTrace();
```
Comment (a-ninja) [inline bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/BigdimeHBaseLogger.java]: remove PST

File: bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/constants/HBaseAlertSchema.java [hunk 4]
```diff
@@ -19,6 +19,8 @@
 		public static final String ALERT_ALERT_MESSAGE = "ms";
 		public static final String ALERT_ALERT_DATE = "dt";
 		public static final String ALERT_ALERT_EXCEPTION = "ex";
+		public static final String ALERT_HOST_NAME = "hn";
+		public static final String ALERT_HOST_IP = "hp";
```
Comment (a-ninja) [inline bigdime-composite-alerting-impl/src/main/java/io/bigdime/alert/multiple/impl/constants/HBaseAlertSchema.java]: use ip as opposed to hp?

**Comment by mnamburi** [comment]
looks good

**Pull request merged**